qt/python/mantidqt/widgets/sliceviewer/test/test_sliceviewer_view.py

```diff
@@ -151,13 +151,6 @@ def test_norm_switches_if_workspace_contains_non_positive_data(self):
         self.assertTrue(isinstance(colorbar.get_norm(), Normalize))
         pres.view.close()
 
-    def test_log_norm_disabled_for_non_positive_data(self):
```

Comment:
This is an excellent example of removing a test to demonstrate expanded behavior